tox.ini

```diff
@@ -3,18 +3,17 @@ envlist = py27, py34
 
 [testenv]
 deps =
-     pip==7.0.1
-     pytest==2.7.1
-     setuptools==16.0
-     wheel==0.24.0
-     flake8==2.4.1
-     py27: mock==1.3.0
+    pip==7.0.1
+    pytest==2.7.1
+    setuptools==16.0
+    wheel==0.24.0
+    flake8==2.4.1
+    numpy==1.9.2
+    scipy==0.16.0
+    pandas==0.16.2
+    scikit-learn==0.16.1
+    py27: mock==1.3.0
 
 commands =
-         pip install numpy --no-index
-         pip install scipy --no-index
-         pip install pandas --no-index
-         pip install scikit-learn --no-index
```

Comment:
You're right: 

 * http://lists.idyll.org/pipermail/testing-in-python/2011-June/004147.html
 * https://github.com/pypa/pip/issues/25
 * https://github.com/scipy/scipy/issues/1954
 * https://github.com/scipy/scipy/pull/453

According to the last link, it should be fixed in `scipy>0.14.0` but I'm reverting to the previous tox config to avoid problems.